docs/guides/jupyter.rst

```diff
@@ -0,0 +1,363 @@
+How to use Jupyter notebooks in Sphinx
+======================================
+
+`Jupyter`_ notebooks are a popular tool to describe computational narratives
```

Comment:
meta point (feel free to disregard) - I am a big fan of [sembr](https://sembr.org/) for deciding when to do line breaks, rather than basing it on character length. If it's too much hassle to think about the "right" place to break lines, I've found that using "one sentence per line" is a nice rubric.

Reply:
Yeah, even if it doesn't look like it, I'm a fan too! However I still haven't mastered the technique, or often put two sentences in one line. Thanks for the reminder though :)